sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample_ModelCompose.md

```diff
@@ -0,0 +1,86 @@
+
+```C# Snippet:FormRecognizerSampleBuildVariousModels
+// For this sample, you can use the training forms found in the `trainingFiles` folder.
+// Upload the forms to your storage container and then generate a container SAS URL.
+// For instructions on setting up forms for training in an Azure Storage Blob Container, see
+// https://aka.ms/azsdk/formrecognizer/buildtrainingset
+
+Uri officeSuppliesUri = <purchaseOrderOfficeSuppliesUri>;
+var officeSupplieOptions = new BuildModelOptions() { ModelDescription = "Purchase order - Office supplies" };
+
+BuildModelOperation suppliesOperation = await client.StartBuildModelAsync(officeSuppliesUri, buildModelOptions: officeSupplieOptions);
+Response<DocumentModel> suppliesOperationResponse = await suppliesOperation.WaitForCompletionAsync();
+DocumentModel officeSuppliesModel = suppliesOperationResponse.Value;
+
+Uri officeEquipmentUri = <purchaseOrderOfficeEquipmentUri>;
+var equipmentOptions = new BuildModelOptions() { ModelDescription = "Purchase order - Office Equipment" };
+
+BuildModelOperation equipmentOperation = await client.StartBuildModelAsync(officeSuppliesUri, buildModelOptions: equipmentOptions);
+Response<DocumentModel> equipmentOperationResponse = await equipmentOperation.WaitForCompletionAsync();
+DocumentModel officeEquipmentModel = equipmentOperationResponse.Value;
+
+Uri furnitureUri = <purchaseOrderFurnitureUri>;
+var furnitureOptions = new BuildModelOptions() { ModelDescription = "Purchase order - Furniture" };
+
+BuildModelOperation furnitureOperation = await client.StartBuildModelAsync(officeSuppliesUri, buildModelOptions: equipmentOptions);
+Response<DocumentModel> furnitureOperationResponse = await furnitureOperation.WaitForCompletionAsync();
+DocumentModel furnitureModel = furnitureOperationResponse.Value;
+
+Uri cleaningSuppliesUri = <purchaseOrderCleaningSuppliesUri>;
+var cleaningOptions = new BuildModelOptions() { ModelDescription = "Purchase order - Cleaning Supplies" };
+
+BuildModelOperation cleaningOperation = await client.StartBuildModelAsync(officeSuppliesUri, buildModelOptions: equipmentOptions);
+Response<DocumentModel> cleaningOperationResponse = await cleaningOperation.WaitForCompletionAsync();
+DocumentModel cleaningSuppliesModel = cleaningOperationResponse.Value;
+```
+
+When a purchase order happens, the employee in charge uploads the document to our application. The application then needs to analyze the document to extract the total value of the purchase order. Instead of asking the user to look for the specific `modelId` according to the nature of the document, you can create a composed model that aggregates the previous models, and use that model in `StartDocumentAnalysis` and let the service decide which model fits best according to the document provided.
```

Comment:
Should this be StartAnalyzeDocument()?

Reply:
Addressed in https://github.com/Azure/azure-sdk-for-net/pull/24340